openeo-geotrellis/src/test/scala/org/openeo/geotrellis/netcdf/NetCDFRDDWriterTest.scala

```diff
@@ -141,6 +142,75 @@ class NetCDFRDDWriterTest extends RasterMatchers{
     assertFalse(isData(rasterValueAt(pointOutsideOfGeometry)))
   }
 
+  @Test
+  def testWriteSamplesItems(): Unit = {
+    val utm31 = CRS.fromEpsgCode(32631)
+    val polygon = MultiPolygon(
+      Polygon(
+        (-180.0, -90.0),
+        (-180.0, 90.0),
+        (180.0, 90.0),
+        (180.0, -90.0),
+        (-180.0, -90.0),
+      ),
+    )
+    val polygons = ProjectedPolygons(polygon,CRS.fromEpsgCode(4326))
```

Comment:
This is OK for a basic test but Is it possible to also test with less ideal samples, e.g. 2 that don't cover the whole world?